content/en/blog/2023/additional-collector-sig.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Additional collector SIGs for APAC
```

Comment:
```suggestion
title: New APAC collector SIG meetings
```

Reply:
```suggestion
title: New APAC Collector SIG meetings
```

Reply:
@MovieStoreGuy - somehow these comments weren't addressed. I'll submit a new PR.

Reply:
Suggestions applied in #2413